src/shared/components/CodeViewToolbar.tsx

```diff
@@ -83,16 +88,18 @@ export class CodeViewToolbar extends React.Component<CodeViewToolbarProps, CodeV
                             </div>
                         )}
                 </ul>
-                <CodeIntelStatusIndicator
-                    key="code-intel-status"
-                    userIsSiteAdmin={currentUser ? currentUser.siteAdmin : false}
-                    repoPath={this.props.repoPath}
-                    commitID={this.props.commitID}
-                    filePath={this.props.filePath}
-                    onChange={this.props.onEnabledChange}
-                    simpleProviderFns={this.props.simpleProviderFns}
-                    site={site}
-                />
+                {!this.state.useExtensions && (
```

Comment:
As mentioned above, could we just do this instead of using storage in this file?

```suggestion
                {!this.props.extensionsController && (
```